src/Math-Core-Distribution/PMNormalDistribution.class.st

```diff
@@ -8,9 +8,22 @@ Class {
 	#classVars : [
 		'NextRandom'
 	],
-	#category : 'Math-Core-Distribution'
+	#category : #'Math-Core-Distribution'
 }
 
+{ #category : #information }
+PMNormalDistribution class >> boxMullerTransform [
```

Comment:
What's interesting is that this is a class method, rather than an instance one. I presume this choice was made because NextRandom is a class variable?

Reply:
no because ``PMNormalDistribution class >> random`` is a class method.